TWLight/static/css/local.css

```diff
@@ -752,3 +753,24 @@ CLEARING COLUMNS
     clear: left;
   }
 }
+
+/* For 'back to top' button on /partners/ */
+#back-to-top {
+  display: inline-block;
+  border-radius: 4px;
+  position: fixed;
+  bottom: 210px;
```

Comment:
I'm going back and forth on whether this should be lower (around 170px). At 170px, the button doesn't cover the footer in any device I have tested it on. Is there a reason why you chose 210px?

Reply:
Reduced the value to 170px you are right it is working fine on this value at all devices :)